bot/cogs/snakes.py

```diff
@@ -16,32 +19,88 @@ def __init__(self, bot: AutoShardedBot):
+                    'description': snake['description'],
+                    'location': snake['location'],
+                    'venomous': snake['venomous'],
+                    'image': snake['image']
+                }
+            else:
+                snake_info = {
+                    'name': snake['name'],
+                    'description': snake['description'],
+                    'creator': snake['creator'],
+                    'created': snake['created'],
+                    'image': snake['image']
+                }
 
-        :param name: Optional, the name of the snake to get information for - omit for a random snake
-        :return: A dict containing information on a snake
-        """
+        return snake_info
 
-    @command()
+    @command(name='get')
     async def get(self, ctx: Context, name: str = None):
         """
-        Go online and fetch information about a snake
+        Shows information on different snakes.
+        """
+
+        snake_info = await self.get_snek(name)
 
-        This should make use of your `get_snek` method, using it to get information about a snake. This information
-        should be sent back to Discord in an embed.
+        embed = Embed(
+            title=snake_info['name'].title(),
+            description=snake_info['description']
+        )
 
-        :param ctx: Context object passed from discord.py
-        :param name: Optional, the name of the snake to get information for - omit for a random snake
+        if snake_info['name'] != "python":
+            embed.add_field(name="Where can you find them?", value=snake_info['location'])
+            embed.add_field(name="Are they venomous?", value=snake_info['venomous'])
+            embed.set_image(url=snake_info['image'])
+        else:
+            embed.add_field(name="Who created it?", value=snake_info['creator'])
+            embed.add_field(name="When was it created?", value=snake_info['created'])
+            embed.set_thumbnail(url=snake_info['image'])
+
+        await ctx.send(embed=embed)
+
+    @command(name='movies')
+    async def movies(self, ctx: Context, movie_name: str = None):
+        """
+        Shows 5 snake movies. Warning: They are all pretty bad.
         """
 
-    # Any additional commands can be placed here. Be creative, but keep it to a reasonable amount!
+        with open('bot/db/movies.json', 'r') as file:
+            movies_dict = json.load(file)
+
+        if movie_name is None or len(movie_name) == 0:
+            embed = Embed(
+                title="Snake Movies",
+                description="A list of snake movies.",
+            )
+
+            for movie in movies_dict.values():
+                embed.add_field(name=movie['title'], value=f"bot.movies('{movie['title'].lower()}')\n\n")
+
+            embed.set_thumbnail(url="https://i.imgur.com/dB38NwN.png")
+
+        elif len(movie_name) > 0:
+            embed = Embed(
+                title=movies_dict[movie_name]['title'],
```

Comment:
This will break if the movie isn't in the dict, or it isn't entirely lowercase. Should fix that.